spec/coverage_helper.rb

```diff
@@ -5,20 +5,19 @@ module CoverageHelper
 
   ##
   # Skip list used to avoid loading certain patterns within
-  # the logstash directories, this patterns are excluded becuause
+  # the logstash directories, this patterns are excluded because
```

Comment:
Dude, you gotta kill whatever is making this change in your editor. `git blame` gets full of lies when this happens :P